Inventory.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InventoryManagement
+{
+    public class Inventory
+    {
+
+        List<Product> products = new List<Product>();
+
+        //Define Constructor 
+        public Inventory() { }
+
+        //Add new Product to the List 
+
+        public void AddProduct(Product product)
+        {
+            products.Add(product);
+        }
+
+        //Delete Product from the List 
+
+        public void RemoveProduct(Product product)
+        {
+            products.Remove(product);
+        }
+
+        //Return All Products in the List 
+        public List<Product> ViewAllProducts()
+        {
+            return products;
+        }
+
+        //Search for a Product in the List using It's Name
+        public Product SearchProduct(String name)
+        {
+
+            foreach (var product in products)
+            {
+
+                if (product.Name.ToLower() == name.ToLower()) return product;
+
+            }
+            return null;
```

Comment:
how can we utilize LINQ here instead of loops?

Reply:
I update it using the FirstOrDefault extension method.